macros/sql/weekdays_between_dates.sql

```diff
@@ -0,0 +1,31 @@
+--code source: https://discourse.looker.com/t/how-to-count-only-weekdays-between-two-dates/3345
+{%- macro weekdays_between_dates(start, stop) -%}
+(
+date_diff('day', {{start}}, {{stop}}) -
```

Comment:
The standard SQL function is `datediff`, not `date_diff`, and while `date_diff` does appear to work on Redshift I don't think it would [on Snowflake](https://docs.snowflake.net/manuals/sql-reference/functions/datediff.html).  And of course BigQuery has to be different, so there `date_diff` would work but `datediff` _wouldn't_!

There is a [`datediff` macro in `dbt_utils`](https://github.com/fishtown-analytics/dbt-utils#datediff-source) that generates the necessary compatible SQL, but this `utilities` package doesn't currently depend on the `dbt_utils` package, and I think there may be complications with `dbt_utils` getting included multiple times in client projects if we try to depend on it here.

So for now my recommendation would be to use the more standard `datediff` SQL function rather than `date_diff`.

Reply:
Switched to datediff